### PR TITLE
Add "useful articles" section and article about codewars

### PR DESCRIPTION
**Pull request by arzucaner**
In this commit, we've introduced a new section called "useful articles" under the "career progression" category. This addition is aimed at fostering community development by providing our members with a platform to share their experiences, guides, and other valuable content. feedback.

Closes issue #77

## Type Of Change
- [x] Update documentation
- [ ] Add new event details
- [x] Add project resources

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Commit message is as per guidelines in the contributor guide

### CODE REVIEW

File: resources/career-progression/2023-09-21-exploring-the-codewars-universe-a-developers-journey.markdown [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#Exploring the Codewars Universe: A Developer’s Journey
+
+##“To solve a problem, madness is required; to investigate, a genius is necessary.” — Sherlock Holmes
+Since my childhood, I have enjoyed solving mysteries. I’ve always read books and watched films in this direction. In fact, I’m a fan of Sherlock Holmes. When curiosity is added to the mystery in my life, solving problems becomes very enjoyable for me.
+
+###When I started as an apprentice at Lonca (Lonca (Guild) is a school of software engineering that follows the software craftsmanship movement, emphasizing an approach to equip students with industry-quality software development skills), I got introduced to Codewars. Because in Lonca’s educational system, you are encouraged and motivated to improve yourself by solving katas on Codewars.
+
+##What is Codewars, and Why Is Solving Problems There Important?
+
+Codewars is a platform for software developers to enhance their coding skills and practice in commonly used programming languages. This platform includes thousands of programming and algorithm questions categorized by various difficulty levels and topics.
```
Comment (dricazenck) [inline resources/career-progression/2023-09-21-exploring-the-codewars-universe-a-developers-journey.markdown:10]: `Codewars` here you could add a link to the page. it would be  nice.

Comment (dricazenck) [reply resources/career-progression/2023-09-21-exploring-the-codewars-universe-a-developers-journey.markdown:10]: also could you add that is similar to leetcode and other platforms. But you can say why do you prefer Codewars.

File: resources/career-progression/2023-09-21-exploring-the-codewars-universe-a-developers-journey.markdown [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+After solving the problem and passing the test, it also provides other solved examples related to that question. You can compare your solution with best practices and get inspiration from other examples. 
+
+##What resources did I use while solving questions?
+
+###For question solutions, I primarily use https://developer.mozilla.org/. If I need another source, I refer to https://stackoverflow.com/.
+If I ever can’t solve a question, I used to take a break for a couple of days and then revisit it. But now, at Lonca, I have my dear friend Taiga. After solving the question, I asked Taiga to show me the options related to that question.
+Taiga generously shares all the information. If I can’t solve a question, I no longer wait. Instead, I immediately asked, “Could you help me with this question that I couldn’t solve?” Taiga provides excellent hints, and if I request, explains the solution methods in detail. It helps me use my time more efficiently and contributes significantly to my improvement. Thanks again, dear friend Taiga! You are a mysterious hero.
```
Comment (dricazenck) [inline resources/career-progression/2023-09-21-exploring-the-codewars-universe-a-developers-journey.markdown:28]: That is a very nice thing to add it, and also please add here Linkedin Taiga

File: resources/career-progression/useful-articles.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+**Description:**
+This added aims to contribute to our community's development. The "Useful Articles" section provides our members with an opportunity to share their experiences, guides, and other helpful content.
+
+**Example:**
+This pull request includes adding an article related to Codewars. This article serves as a guide that can help our members enhance their programming skills and be better prepared for job interviews.
+
+**We Welcome Your Participation:**
+We encourage other members to share their own articles and content in this section. Please review this change and share your feedback.
+
+By approving this change, we can add a valuable resource to our community. Thank you!
```
Comment (dricazenck) [inline resources/career-progression/useful-articles.md:12]: ❤️

**Comment by dricazenck** [comment]
I really liked the idea for this useful article!  Thank you for your contribution. 

I think you could submit this blog post to global, and we can also share it on our social media later.

File: README.md [hunk 4]
```diff
@@ -340,7 +340,7 @@ We are looking for volunteers, and we prepared a detailed and comprehensive guid
 
 ## About Women Who Code
 
-WWCode is the largest and most active community of technical women in the world. We have over 320,000 members in 147 countries & counting. Join our community by visiting [womenwhocode.com](womenwhocode.com) and sign up to become a member.
+WWCode is the largest and most active community of technical women in the world. We have over 343,000 members in 147 countries & counting. Join our community by visiting [womenwhocode.com](womenwhocode.com) and sign up to become a member.
```
Comment (dricazenck) [inline README.md:343]: Considering the focus for your PR here is the usefull articles, I do recommend you to create another PR just for this change on README. It is always a good practice to create small PR's and in the title you could describe that this solves the issue you just created.

**Comment by irenkamalova** [comment]
Hi @arzucaner!
This article is great!
Why don't you want to share it though blog post with WWCode?

File: resources/career-progression/2023-09-21-exploring-the-codewars-universe-a-developers-journey.markdown (hunk 1 above)
Comment (arzucaner) [reply resources/career-progression/2023-09-21-exploring-the-codewars-universe-a-developers-journey.markdown:10]: In fact, I first encountered Codewars when I started my software development training, and I continued my studies from there. I had published this article on my Medium account. I shared it here for the sake of example. Before sharing it here, I did some research and learned that I needed to add it in markdown format. Then, I prepared my writing to be suitable for markdown and added it. I think there might have been a mistake in this part. It might have been more appropriate to open this directly as a PR request and include the link. Mistakes are the best way to learn. I will try this again. Thank you.

File: README.md (hunk 4 above)
Comment (arzucaner) [reply README.md:343]: Thank you very much for this helpful feedback.

**Comment by arzucaner** [comment]
I had shared this article on my Medium account. I'm not sure if they will accept it since it was published before. But if they find it suitable, I would love to share it.

**Comment by irenkamalova** [comment]
It's true, they won't accept published articles. 
Thank for sharing this one here! ❤️

**Review by irenkamalova — APPROVED** [review]

File: resources/career-progression/2023-09-21-exploring-the-codewars-universe-a-developers-journey.markdown (hunk 1 above)
Comment (dricazenck) [reply resources/career-progression/2023-09-21-exploring-the-codewars-universe-a-developers-journey.markdown:10]: no, it is totally fine. You didn't do anything wrong. There are many ways to do things. 😄  I just added some suggestions for your article.

**Review by dricazenck — APPROVED** [review]
